econml/tests/test_causal_analysis.py

```diff
@@ -481,3 +490,86 @@ def test_over_cat_limit(self):
 
         # column d is now okay, too
         self.assertEqual([res.feature_name for res in ca._results], ['a', 'b', 'c', 'd', 'f', 'g', 'h'])
+
+    def test_individualized_policy(self):
+        y = pd.Series(np.random.choice([0, 1], size=(500,)))
+        X = pd.DataFrame({'a': np.random.normal(size=500),
+                          'b': np.random.normal(size=500),
+                          'c': np.random.choice([0, 1], size=500),
+                          'd': np.random.choice(['a', 'b', 'c'], size=500)})
+        inds = ['a', 'b', 'c', 'd']
+        cats = ['c', 'd']
+        hinds = ['a', 'd']
+
+        ca = CausalAnalysis(inds, cats, hinds, heterogeneity_model='linear')
+        ca.fit(X, y)
+        df = ca.individualized_policy(X, 'a')
+        self.assertEqual(df.shape[0], 500)  # all rows included by default
+        self.assertEqual(df.shape[1], 4 + X.shape[1] - 1)  # new cols for policy, effect, upper and lower bounds
+        df = ca.individualized_policy(X, 'b', n_rows=5)
+        self.assertEqual(df.shape[0], 5)
+        self.assertEqual(df.shape[1], 4 + X.shape[1] - 1)  # new cols for policy, effect, upper and lower bounds
+        # verify that we can use a scalar treatment cost
+        df = ca.individualized_policy(X, 'c', treatment_costs=100)
```

Comment:
Will you standardize `treatment_cost` vs `treatment_costs`? I think this class originally used `treatment_cost` which has a nice vector-value interpretation. If the default is `0` with optional individualized cost, then the singular "cost" makes sense to me.

Reply:
Good point, thanks, I'll switch it to `treatment_cost` here, too.

Reply:
Er, actually I'm going to do the opposite and standardize on `treatment_costs` because that's what's already used in our interpreters, and if we have multiple discrete treatments then there really can be multiple costs per sample.

Reply:
Makes sense! Thanks